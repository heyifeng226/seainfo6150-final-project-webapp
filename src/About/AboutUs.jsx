import React from "react";
import { Link } from "react-router-dom";
import styles from "./AboutUs.module.css";

const AboutUs = () => {
  return (
    <section className={styles.aboutContainer}>
        <div className={styles.About_part}>
            <h2>About Us</h2>
            <img
                src="https://lh3.googleusercontent.com/proxy/JBXEaT1IMMMjJSyMxp8EP2YuwGoijv9tjI8omDfksPcxdalG6EDFPL7q3ielZWsIGN0qP_McnO6DEvQ7r2rg9MCgClu5B0I"
                alt="Company"/>
            <p>Cooking Smarter. Together.
                Myrecipes is an original and food-focused social network created for cooks by cooks; 
                where everyone plays a part in helping cooks discover and share the joy of cooking
            </p>
        </div>

        <div className={styles.Team_part}>
            <h2>Our Teams</h2>
            <img
                src="https://www.careeraddict.com/uploads/article/58721/illustration-group-people-team-meeting.jpg"
                alt="Team"/>
            <p>The professional team of the MyRecipes website was established in
                2020. 
            </p>
        </div>

        <div className={styles.Careers_part}>
            <h2>Careers</h2>
            <img
                src="https://nextadvocate.com/wp-content/uploads/2019/03/Careers-e1553244124511.jpg"
                alt="Career"/>
            <p>Dreamers Wanted:<br/>
                Our amazing team of people, the very heart of our company, breathe life into our
                vision every day with their ingenuity and collaborative spirit. Dont' wait to join us!
            </p>
        </div>
    </section>
  );
};

export default AboutUs;
