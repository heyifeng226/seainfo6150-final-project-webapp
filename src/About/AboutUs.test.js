import React from "react";
import { render } from "@testing-library/react";
import AboutUs from "./AboutUs";


describe("AboutUs tests", () => {
    it("renders correctly", () => {
      const { container } = render(<AboutUs />);
      expect(container).toMatchSnapshot();
    });
  
  
  });