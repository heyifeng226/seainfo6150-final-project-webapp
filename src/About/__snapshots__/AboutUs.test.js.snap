// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs tests renders correctly 1`] = `
<div>
  <section
    class="aboutContainer"
  >
    <div
      class="About_part"
    >
      <h2
        class="title"
      >
        About Us
      </h2>
      <img
        alt="About Us"
        src="https://www.careeraddict.com/uploads/article/58721/illustration-group-people-team-meeting.jpg"
      />
      <p>
        Hello and welcome to MyRecipes! Thank you so much for stopping by the site! If you are new to MyRecipes, the one thing you should know about us is that we are obsessed with creating scratch cooking recipes that you will love.
      </p>
      <p>
        MyRecipes was founded in 2020, and was produced by a team of experienced cooks devoted to high quality, accessible home cooking. MyRecipes is an original and food-focused social network created for cooks by cooks; where everyone plays a part in helping cooks discover and share the joy of cooking
      </p>
      <p>
        Our goal is to encourage people to cook at home, and to make the process of feeding your family and loved ones less intimidating and more enjoyable. Our recipes are all tested in our own home kitchens, usually several times.
      </p>
      <p>
        Cooking Smarter. Together.
      </p>
      <p>
        Thanks so much for visiting MyRecipes!
      </p>
    </div>
  </section>
</div>
`;
